lib/web/ui/app.go

```diff
@@ -199,17 +202,27 @@ func makePermissionSets(src []*types.IdentityCenterPermissionSet) []IdentityCent
 // Web UI. Thus, this field is currently not available in the Connect App type.
 func MakeAppTypeFromSAMLApp(app types.SAMLIdPServiceProvider, c MakeAppsConfig) App {
 	labels := ui.MakeLabelsWithoutInternalPrefixes(app.GetAllLabels())
+	uiLaunchURLs := func(in []string) []SAMLAppLaunchURL {
+		out := make([]SAMLAppLaunchURL, 0, len(in))
+		for _, u := range in {
+			out = append(out, SAMLAppLaunchURL{
+				URL: u,
+			})
```

Comment:
should we be using the `FriendlyName` field anywhere here? I don't think i see it used, curious if it might be vestigial from a previous iteration.

Reply:
`FriendlyName` is not implemented in the backend yet. I only realized I should have supported that in my first attempt. But I defined that in the web API response just so it self-aligns the response shape if I ever update the backend to support it.